flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/DockerImageVersions.java

```diff
@@ -42,7 +42,7 @@ public class DockerImageVersions {
 
     public static final String LOCALSTACK = "localstack/localstack:0.13.3";
 
-    public static final String PULSAR = "apachepulsar/pulsar:2.10.2";
+    public static final String PULSAR = "apachepulsar/pulsar:2.11.1";
```

Comment:
Maybe you could set Luna Streaming 2.10_4.4 for the docker image

Reply:
Yes. Good idea

Reply:
I'll do it in a subsequent PR.
So tis one can be contributed upstream